pkg/k8shandler/index_management.go

```diff
@@ -0,0 +1,128 @@
+package k8shandler
+
+import (
+	"fmt"
+	"strings"
+
+	logging "github.com/openshift/elasticsearch-operator/pkg/apis/logging/v1"
+	"github.com/openshift/elasticsearch-operator/pkg/indexmanagement"
```

Comment:
there's a lot of `indexManagement` in this file which makes it confusing at first glance to discern what we are referencing.. can we add a name to this import like `elasticIndexManagement`?

Reply:
would you be agreeable to something like `im` which is shorter.  It's already the "ElasticSearch" operator is there need to reinforce that in the package alias?

Reply:
`idxmgmt` is relatively short, and relatively legible

Reply:
updated to use `spec` for the object; `indexmanagement.VerifyAndNormalize(cluster)` reads well